export/pdf-component.js

```diff
@@ -119,6 +119,11 @@ class PrintToPdf extends HTMLElement {
           scrollX: 0,
           scrollY: 0,
           scale: event.detail?.quality || 4
+        },
+        jsPDF: {
+          orientation: event.detail?.orientation || 'portrait',
+          unit: event.detail?.unit || 'in',
+          format: event.detail?.format || 'letter'
```

Comment:
Why don't we default to cm and A4? It feels weird because the old reports fitted a A4. Maybe we told jsPDF to do letterbox but then we fixed it with the @page on the css?

Reply:
I just add to default the default values the library provides, as we didn't set this before was using the default values so I keep it as it is, we can define our defaults if it is what you mean.